packages/translate/lib/testLine.tsx

```diff
@@ -89,11 +89,15 @@ const translateAssertTestLine = (
 	elements?: Elements,
 	lineResult?: TestLineResult,
 ): TestLineNode => {
+	let status = lineResult?.result;
+	if (testLine.then !== 'success' && status) {
+		status = mapStatus(lineResult?.result) === 'success' ? testLine.then : 'success';
```

Comment:
`...mapStatus(status)...`? Because `let status = lineResult?.result`.
Small thing, but it's actually transpiled to a long string by TypeScript when compiled and this things add up.